app/Migrations/Version20200602134547.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace CultuurNet\UDB3\Silex\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20200602134547 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $table = $schema->createTable('productions');
+        $table->addColumn(
+            'event_id',
+            'guid',
+            [
+                'length' => 36,
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'production_id',
+            'guid',
+            [
+                'length' => 36,
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'name',
+            'string',
+            [
+                'length' => 32,
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'added_at',
+            'date_immutable',
+            [
+                'notnull' => true,
+            ]
+        );
+
+        $table->setPrimaryKey(['event_id']);
+        $table->addIndex(['production_id']);
```

Comment:
Thinking a bit ahead about the search functionality on name, should we also add an index on `name`?
The `MATCH()` function looks interesting: https://dev.mysql.com/doc/refman/8.0/en/fulltext-natural-language.html

Reply:
Or if you want to add that as another migration in a later PR, that's also fine 👍

Reply:
> Full-text indexes can be used only with InnoDB or MyISAM tables, and can be created only for CHAR, VARCHAR, or TEXT columns.

Do you know what engine we are using?

Reply:
Local, acceptance and test all use InnoDB. Production I'm not 100% sure, but I think we can safely assume it should be the same as acceptance and test.